bigtable/tests/unit/test_row.py

```diff
@@ -164,23 +149,28 @@ def test_set_cell_with_non_null_timestamp(self):
         from google.cloud._helpers import _EPOCH
 
         microseconds = 898294371
-        millis_granularity = microseconds - (microseconds % 1000)
         timestamp = _EPOCH + datetime.timedelta(microseconds=microseconds)
-        self._set_cell_helper(timestamp=timestamp,
-                              timestamp_micros=millis_granularity)
+        self._set_cell_helper(timestamp=timestamp)
 
     def test_delete(self):
+        from google.cloud.bigtable_v2.proto import bigtable_pb2
+
         row_key = b'row_key'
         row = self._make_one(row_key, object())
-        self.assertEqual(row._pb_mutations, [])
+        entry = bigtable_pb2.MutateRowsRequest.Entry()
+        self.assertEqual(row.row_mutations.mutations_entry.mutations,
+                         entry.mutations)
         row.delete()
 
         expected_pb = _MutationPB(
             delete_from_row=_MutationDeleteFromRowPB(),
         )
-        self.assertEqual(row._pb_mutations, [expected_pb])
+        self.assertEqual(row.row_mutations.mutations_entry.mutations[0],
+                         expected_pb)
```

Comment:
This assertion allows for (unwanted) additional mutations.  It either needs an additional check of the length, or else to be inverted, e.g.:

```python
        self.assertEqual(row.row_mutations.mutations_entry.mutations,
                         [expected_pb])
```

Reply:
@tseaver I am getting error ```TypeError: Can only compare repeated composite fields against other repeated composite fields.```. I think it is because mutations is the protobuf message

Reply:
@aneepct can you convert `expected_pb` into a MutationEntry?

Reply:
@sduskis You just need to change the name? or You want to compare Entry from Mutation? Here I am checking that which mutation is called when row.delete() api called.